lib/core/shared/sdk/embeddedSdk.ts

```diff
@@ -36,13 +37,39 @@ import ImpersonatedSDK from './impersonatedSdk';
 
 const contextError = kerror.wrap('plugin', 'context');
 
+const deprecatedForbiddenEmbeddedActions = {
+  'auth': new Set([
```

Comment:
Why using a Set here? A simple array will also works and it's more "portable"

Reply:
Because it is an unordered collection of unique elements, thus a set. For portability, Set is a native javascript object

Reply:
Good point :+1: Since the initialization is not that different from native arrays it's ok to me